Alignment/OfflineValidation/python/TkAlAllInOneTool/dataset.py

```diff
@@ -349,7 +348,7 @@ def __getData( self, dasQuery, dasLimit = 0 ):
         except KeyError:
             error = None
         if error or self.__findInJson(jsondict,"status") != 'ok' or "data" not in jsondict:
-            jsonstr = str(jsondict)
+            jsonstr = jsondict['reason']
```

Comment:
@vbotta Looking at the condition `"data" not in jsondict` it might be that the handling is not quite correct here.
The "reason" key is probably only present if DAS reported an error it is aware of. It looks like it can happen that DAS reports no error, but provides "data" key.
It is probably safest here to check if the "reason" key is there and do what you are suggesting here. If it is missing, just print the dictionary (as was done previously).